Tasks/DockerV0/utils.ts

```diff
@@ -82,6 +85,28 @@ export function getImageMappings(connection: ContainerConnection, imageNames: st
     return sourceToTargetMapping;
 }
 
+
+function getTaskOutputDir(command: string): string {
```

Comment:
This section was copied from the Utils file from DockerV2 I would like to know if there is a process to share functions between the different tasks.